doc/external-libs/psblas.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "https://www.w3.org/TR/html4/loose.dtd">
+<html>
+
+<head>
+  <title>The deal.II Readme on interfacing to PSBLAS</title>
+  <link href="../screen.css" rel="StyleSheet">
+  <meta name="copyright" content="Copyright (C) 2025 by the deal.II authors">
+  <meta name="keywords" content="deal.II">
+</head>
+
+<body>
+
+  <h1>Interfacing <acronym>deal.II</acronym> to psblas</h1>
```

Comment:
They write it as "PSBLAS", so I think you should use all-caps here and in all the other places.